gridcapa-task-manager-app/src/main/java/com/rte_france/farao/gridcapa_task_manager_app/TaskManagerApplication.java

```diff
@@ -16,6 +17,7 @@
 @SuppressWarnings("hideutilityclassconstructor")
 @SpringBootApplication
 @EnableConfigurationProperties(TaskManagerConfigurationProperties.class)
+@EntityScan(basePackages = "api.entities")
```

Comment:
What is that for ?

Reply:
entities used by spring are defined in another package because they need to be in the api. I could not get the application work without adding this annotation; for more information see https://www.baeldung.com/spring-entityscan-vs-componentscan#the-entityscan-annotation for example